_component_design/radio-buttons.md

```diff
@@ -7,157 +7,140 @@ usage: >
   Radio buttons are used when the user is only allowed to select one option from a list. A minimum of one choice is required, and one is preselected by default.
 preview-image: preview-images/radio-buttons.svg
 resource: true
-status: in-progress
+status: stable
 need: selectors
-last-modified: 2017-08-17
+last-modified: 2017-11-04
 ---
 
 {% include toc.html %}
 
-<!--{% include todo.html content="Update focus states, expand on inline validation,
-and improve keyboard accessibility." %}-->
 
-# Radio Buttons
 
-Use when selecting a **single item** from a set it's important for the user to
-see all options at once. If viewing items side-by-side isn't important,
-consider a selector, which uses less space.
+
+
+
+## Introduction
 
 <div class="hxRow">
-{% column left:"hxCol" %}
-## Enabled
-
-<div class="ui form">
-  <div class="grouped fields">
-    <label>Select an option</label>
-    <div class="field">
-      <div class="ui radio checkbox">
-        <input type="radio" name="radio1" checked="checked">
-        <label>Option 1</label>
-        <div class="ui message subtext">
-          This is some subtext.
-        </div>
-      </div>
-    </div>
-    <div class="field">
-      <div class="ui radio checkbox">
-        <input type="radio" name="radio1">
-        <label>Option 2</label>
-      </div>
-    </div>
-    <div class="field">
-      <div class="ui radio checkbox">
-        <input type="radio" name="radio1">
-        <label>Option 3</label>
-      </div>
-    </div>
-  </div>
-</div>
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+### When to use
+
+- Radio buttons should be used when a user has to select **only one option** in a list.
+- If multiple options can be selected, use [checkboxes]({{site.baseurl}}/components/checkboxes.html) instead.
+- A [dropdown menu]({{site.baseurl}}/components/dropdowns.html) can also be used for selecting only one option, but radio buttons are a better pattern if you want to reveal all options on the page without asking the user to click on the dropdown to see all available options. If you’re using more than five options, strongly consider the dropdown pattern.
```

Comment:
only one option. Radio buttons are...to click on the dropdown.
s/reveal/show

Reply:
updated.